### PR TITLE
feat: component css property ordering

### LINKED ISSUE #55: Component CSS property ordering
It can be annoying when added CSS properties jump to the bottom after saving.

Try to find a solution that doesn't require changing the site data structure.

### PR DESCRIPTION
**Pull request by abemscac**
Close #55

### CODE REVIEW

File: libs/frontend/ui-widgets/src/lib/form/PSMultiselect.vue [hunk 1]
```diff
@@ -23,7 +23,7 @@
         autocomplete="off"
         @update:modelValue="updateSearch"
         @keydown.enter.stop="searchEnter"
-        @keydown.enter.tab="searchEnter"
+        @keydown.tab.stop="searchEnter"
```
Comment (abemscac) [inline libs/frontend/ui-widgets/src/lib/form/PSMultiselect.vue:26]: Fixed a bug where select event will be emitted twice when enter is pressed.

https://github.com/samatechtw/pubstudio-builder/assets/59676941/3ef276aa-afff-47cc-ba76-51e8a0930a25

File: libs/frontend/util-render/src/lib/util-render.ts [hunk 2]
```diff
@@ -18,7 +18,8 @@ export const rawStyleRecordToString = (
     const fields = Object.entries(rawStyle)
       .map(([prop, value]) => {
         const computedValue = computeCssValue(prop as Css, value, context)
-        return `${prop}:${computedValue};`
+        const supportedValue = CSS.supports(prop, computedValue) ? computedValue : ''
+        return `${prop}:${supportedValue};`
```
Comment (abemscac) [inline libs/frontend/util-render/src/lib/util-render.ts:22]: Fixed a bug where page layout will act strangely when invalid CSS value containing `{`, `}`, `[` shows up.

https://github.com/samatechtw/pubstudio-builder/assets/59676941/98650207-cbda-4ee7-9f31-b51b3e691cd3

File: libs/frontend/ui-widgets/src/lib/form/PSMultiselect.vue (hunk 1 above)
Comment (sampullman) [reply libs/frontend/ui-widgets/src/lib/form/PSMultiselect.vue:26]: There's still some odd behavior with multiselect, but I think I can fix it in #147

**Comment by sampullman** [comment]
This works for now! I'll create a backlog issue for custom ordering later.

**Pull request merged**